src/pages/index.vue

```diff
@@ -51,9 +53,20 @@ const props = defineProps({
         type: String,
         required: false,
         default: ""
+    },
+    adminLevel: {
+        type: String,
+        required: false,
+        default: ""
     }
 });
 
+const getRouterLink = (name: string) => {
```

Comment:
This could potentially be a more flexible util (or maybe a couple of utils_ which could be used by other  components which build routes e.g. Choropleth - would need to include options for either setting a new selected country or keeping the existing one though...

Reply:
for now left this because we no longer need this function since the default is to add admin2 to the route if not specified